pytest_plugins/markers.py

```diff
@@ -14,6 +14,7 @@ def pytest_configure(config):
         "destructive: Destructive tests",
         "upgrade: Upgrade tests",
         "e2e: End to end tests",
+        "stream: Tests unique to stream builds; purged when robottelo is branched.",
```

Comment:
Would you explain in terms of automation how would we purge the `stream` marker on master is branched?

Are we have to manually remove all `stream` markers manually?

If yes, isnt this a very manual and pathetic process we are offering in automation ? Whos responsible for purging?

Reply:
It wouldn't have to manual. A single `sed` command can remove them all, but yes they would need to be removed before branching. It would likely be a good idea to add a `create-branch` type script for all of the manual changes we're also doing around the repo.

Reply:
@JacobCallahan Yeah, the plan is to auto-branch :) !